package.json

```diff
@@ -207,7 +207,7 @@
     "test:js:watch": "npm run test:js -- --watch",
     "test:karma": "npm-run-all --parallel test:karma:*",
     "test:karma:edit-story": "karma start karma-edit-story.config.cjs --single-run",
-    "test:karma:edit-story:watch": "karma start karma-edit-story.config.cjs",
+    "test:karma:edit-story:watch": "karma start karma-edit-story.config.cjs --debug",
```

Comment:
Should we remove it? (you can always add that when running the command if needed)